public/styles/SearchPage.scss

```diff
@@ -11,28 +11,33 @@
           text-transform: capitalize;
         }
 
-        select {
-          -webkit-appearance: none;
-          background-color: $white-color;
-          border-color: $filter-border;
-          -webkit-border-radius: 0;
-          border-radius: 0;
-          display: block;
-          font-size: 0.7em;
-          margin: 0 0 1em;
-          padding: 0.9em 0.4em;
-          text-transform: lowercase;
-          width: 100%;
-        }
-
         #authors-options {
           text-transform: capitalize;
         }
+
+        .css-10nd86i {
+          margin: 0 0 1em;
+
+          .css-vj8t7z, .css-2o5izw, .css-15k3avv {
+            border-color: $filter-border;
+            border-radius: 0;
+            font-size: 0.7em;
+
+            .css-1492t68, .css-d8oujb {
```

Comment:
Can you use a meaningful scss class name?  I feel `css-1492t68` is not meaningful.

Reply:
These class names come with the `react-select` package and were not chosen by me, therefore cannot be changed.